CommunityFridgeMapApi/functions/image/v1/app.py

```diff
@@ -0,0 +1,45 @@
+import json
+import base64
+
+try:
+    from storage import Storage
+except:
+    from dependencies.python.storage import Storage
+
+class ImageHandler:
+    @staticmethod
+    def get_binary_body_from_event(event: dict) -> bytes:
+        """Extract binary data from request body"""
+        assert event["isBase64Encoded"]
```

Comment:
Making sure the request body is a base64 encoded binary data.
When user uploads a raw binary file, AWS lambda encodes the data in base64 before passing it to the `lambda_handler`. That's why we need to decode the data first before writing it in S3.